packages/backend/src/nest/common/logger.ts

```diff
@@ -0,0 +1,9 @@
+import { logger } from '@quiet/logger'
+
+const createLogger = (name: string) => {
+  return {
+    log: logger('backend')(name),
```

Comment:
maybe just 
```
return logger('backend')(name)
```
?

I've seen in many places things linke `log.log.error(...)` as a result of this